packages/background/src/utils/rateService.ts

```diff
@@ -53,10 +69,12 @@ export const chainLinkService: RateService = {
     },
 };
 
-export const coingekoService: RateService = {
+export const coingekoService: (endpoint: string) => CoingekoService = (
+    baseEndpoint: string
```

Comment:
can we add the public endpoint as a backup endpoint and make a call to it here https://github.com/block-wallet/extension/blob/fd26fef13773f3fe144949f4e16ce3a69df2654f/packages/background/src/utils/rateService.ts#L93 when there's a 429 or 5xx error?

Reply:
Yes. Also if our service is not working properly. Requests to chain-fee use a fallback system too

Reply:
Added fallback request 👍